groups/groups.yaml

```diff
@@ -372,6 +372,22 @@ groups:
     members:
       - bentheelder@google.com
 
+  - email-id: k8s-infra-staging-sig-storage@kubernetes.io
+    name: k8s-infra-staging-sig-storage
+    description: |-
+      ACL for staging sig storage artifacts
+    settings:
+      ReconcileMembers: "true"
+    members:
+      - davanum@gmail.com
+      - saadali@google.com
+      - msau@google.com
+      - xingyang105@gmail.com
+      - jsafrane@redhat.com
+      - mattwon@amazon.com
+      - kiran.mova@mayadata.io
+      - humble.devassy@gmail.com
```

Comment:
Please include me = patrick.ohly@intel.com. I was also in k8s-infra-push-csi@kubernetes.io.